### PR TITLE
[eas-cli] add `build:create` command

### PR DESCRIPTION
**Pull request by wkozyra95**
build command from expo-cli without project configuration and `credentials.json` support`

### CODE REVIEW

File: packages/eas-cli/src/build/create.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
+import { EasJsonReader } from '@eas/config';
```
Comment (dsokal) [inline packages/eas-cli/src/build/create.ts:1]: The filename is confusing to me.

**Review by dsokal — APPROVED** [review]

File: packages/eas-cli/src/project/ensureProjectExists.ts [hunk 2]
```diff
@@ -92,7 +92,7 @@ async function registerNewProjectAsync({
       json: {
         accountName,
         projectName,
-        privacy: privacy ?? ProjectPrivacy.PUBLIC,
+        privacy: privacy ?? 'public',
```
[the remark this answers is not in the document]
Comment (wkozyra95) [reply packages/eas-cli/src/project/ensureProjectExists.ts:95]: ProjectPrivacy does not support unlisted

File: packages/eas-cli/src/build/create.ts (hunk 1 above)
Comment (wkozyra95) [reply packages/eas-cli/src/build/create.ts:1]: it's an implementation of `build:create` command

File: .eslintignore [hunk 3]
```diff
@@ -1,2 +1,6 @@
-build
+**/build/**
+
+# re-include ./packages/eas-cli/src/commands/build and ./packages/eas-cli/src/build
+!**/src/**/build/**
```
Comment (dsokal) [inline .eslintignore:4]: So maybe this would be sufficient?
```
/packages/*/build
```

**Review by barthap — APPROVED** [review]

**Pull request merged**